pom.xml

```diff
@@ -11,24 +11,21 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <!--Need to be set in settings.xml to deploy artifacts-->
-    <!--<gpg.keyname>""</gpg.keyname>-->
-    <!--<gpg.passphrase>""</gpg.passphrase>-->
   </properties>
 
   <scm>
     <connection>scm:git:git@github.com:LiveRamp/pom-common.git</connection>
     <url>scm:git:git@github.com:LiveRamp/pom-common.git</url>
     <developerConnection>scm:git:git@github.com:LiveRamp/pom-common.git</developerConnection>
-    <tag>pom-common-1.2</tag>
+    <tag>pom-common-1.1</tag>
   </scm>
 
   <description>pom-common is the common POM for LiveRamp's open-source projects</description>
 
   <developers>
     <developer>
-      <name>Ben Podgursky</name>
-      <email>bpodgursky@gmail.com</email>
+      <name>LiveRamp Engineering</name>
+      <email>info@liveramp.com</email>
```

Comment:
This will go the right place?

Reply:
not really -- i sort of want it to be one level of indirection away from actual engineering. I know the email does work as it is listed on our public website.